src/main/java/com/jfrog/ide/common/go/GoTreeBuilder.java

```diff
@@ -68,7 +68,7 @@ public DepTree createDependencyTree(GoDriver goDriver, Log logger, boolean verbo
         } catch (IOException e) {
             // Errors occurred during running "go list". Run again and this time ignore errors.
             usedModulesResults = goDriver.getUsedModules(false, true, dontBuildVcs);
-            logger.warn("Errors occurred during building the Go dependency tree. The dependency tree may be incomplete:" +
+            logger.error("Errors occurred during building the Go dependency tree. The dependency tree may be incomplete:" +
```

Comment:
I think it is more warning than error because the user will still see a dependency tree